Sources/Private/Experimental/ExperimentalAnimationLayer.swift

```diff
@@ -142,69 +142,7 @@ final class ExperimentalAnimationLayer: CALayer {
 
 }
 
-// MARK: - CALayer + setupLayerHierarchy
-
-extension CALayer {
-  /// Sets up an `AnimationLayer` / `CALayer` hierarchy in this layer,
-  /// using the given list of layers.
-  func setupLayerHierarchy(
```

Comment:
Moved this to its own file (The new implementation is completely different from this old implementation though)